.github/workflows/ci.yaml

```diff
@@ -13,7 +13,7 @@ env:
   THREADS: 4
   CONFIG: RelWithDebInfo
   ALPAKA_BRANCH: 0.9.0
-  VCPKG_INSTALL: "vcpkg install fmt tinyobjloader boost-mp11 boost-atomic boost-smart-ptr boost-functional boost-container; vcpkg install --head xsimd"
+  VCPKG_INSTALL: "vcpkg install fmt tinyobjloader boost-mp11 boost-atomic boost-smart-ptr boost-functional boost-container; until vcpkg install --head xsimd; do echo 'Retrying'; done"
```

Comment:
might this cause an infinite loop?

Reply:
AFAIK the github action will be terminated after some time if this never succeeds.